app/assets/stylesheets/content/_widget_box.sass

```diff
@@ -112,25 +112,23 @@ $widget-box--enumeration-width: 20px
   font-weight: bold
   font-size: 1.25rem
   display: flex
+  margin-top: 4px
+  margin-bottom: 13px
+  border-bottom: none
+  padding-bottom: 0
 
 .widget-box--header-title
   vertical-align: middle
   margin-bottom: 0
   flex-grow: 1
   align-self: center
 
-  .widget-box--header-icon + &
-    padding-left: 5px
-
   // Ensure style for read-only widget headers
   .editable-toolbar-title--fixed
     color: #5F5F5F
-    font-size: 20px
-    font-weight: normal
-
-.widget-box--header-icon
-  align-self: center
-  line-height: 28px
+    font-size: 16px
```

Comment:
As this are the same styles you apply for the input field [here](https://github.com/opf/openproject/blob/e5afd2f1cc3d9d74ccfaecd651d77ddfab62e5dc/app/assets/stylesheets/content/_grid.sass#L121) it makes sense to create a mixing for this. Thus we can be sure that both, editable and non-editable headers, will always look and feel the same.

Reply:
I united them.